generators/structure-component/templates/component.class.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { compose } from 'redux';
+
+import { Wrapper } from './<%= name %>.style';
+
+export namespace <%= name %> {
+  export type StatProps = {};
+
+  export type DispatchProps = {};
+
+  type OwnProps = {};
+
+  export type Props = OwnProps & StatProps & DispatchProps;
+}
+
+class <%= name %>Pure extends React.Component<<%= name %>.Props> {
```

Comment:
nie chcemy mieć czase suffixu Dumb zamiast Pure, jako że to nie jest w sumie Pure?